examples/experimental/persistent-volume-provisioning/README.md

```diff
@@ -0,0 +1,138 @@
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/examples/experimental/persistent-volume-provisioning/README.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+## Persistent Volume Provisioning
+
+This example shows how to use experimental persistent volume provisioning.
+
+### Pre-requisites
+
+This example assumes that you have an understanding of Kubernetes administration and can modify the scripts that launch kube-controller-manager.
+
+### Admin Configuration
+
+Admins configure their clusters with Provisioners tied to quality-of-service tiers.  The correct configuration will be
+the tier name and the volume plugin responsible for provisioning that request separated by a slash.  The configuration is
+added as a CLI flag to kube-controller-manager.
+
+For example, an admin running in AWS can add tier "foo" that provisioned using EBS:
+
+`--qos-tier="foo/kubernetes.io/aws-ebs"`
+
+The CLI flag "--qos-tier" can be used many times to add many provisioners. As the number and types of provisioners increase, admins
+can add many QoS tiers with various backends to satisfy their needs.
+
+```
+--qos-tier="foo/kubernetes.io/aws-ebs" 
+--qos-tier="gold/kubernetes.io/aws-ebs-ssd" # future implementation
+--qos-tier="silver/kubernetes.io/aws-ebs-hd" # future implementation
+--qos-tier="bronze/kubernetes.io/aws-ebs-glacial" # future implementation
+```
+
+#### Provisioners
+
+The following basic provisioners are implemented in Kubernetes:
+
+```
+--qos-tier="foo/kubernetes.io/aws-ebs" 
+--qos-tier="bar/kubernetes.io/gce-pd
+--qos-tier="baz/kubernetes.io/cinder"
+--qos-tier="testing/kubernetes.io/host-path"
+```
+
+Note that the first three are mutually independent and only 1 would be available at a time in a cloud environment.
+The HostPath provisioner is for testing and development purposes only. It *will not work* in a multi-node environment.
+It is unsupported and should not be used for anything other than testing and development.
+
+
+### User storage requests
+
+Users request dynamically provisioned storage by including a quality-of-service tier in their `PersistentVolumeClaim`.
+An annotation is used to access this experimental feature.  In the future, the QoS tier request will likely be a field on the claim itself.
+
+Assuming an admin configures a "foo" provisioner, a user would create a PVC with a special annotation requesting a quality-of-service tier:
+
+```
+{
+  "kind": "PersistentVolumeClaim",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "claim1",
+    "annotations": {
+      "volume.experimental.kubernetes.io/quality-of-service": "foo"
```

Comment:
is this an experimental feature meaning we won't provide migration?  or is this a beta feature now?